week5/MealsharingNET/Controllers/MealController.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.AspNetCore.Mvc;
+using MealsharingNET;
+namespace MealsharingNET.Controllers;
+
+
+
+[ApiController]
+[Route("Meals")]
+public class MealController : ControllerBase
+{
+    private IMealRepository _repo;
+
+    public MealController(IMealRepository repo)
+    {
+        _repo = repo;
+    }
+    /* All meals in list*/
+    
+    [HttpGet("MealList")]
+    public async Task<IEnumerable<Meal>> MealsList()
+    {
+
+        return await _repo.MealsList();
+    }
+
+    /* Add a meal in meal list*/
+
+    [HttpPost("AddMeal")]
+    public void AddMeal([FromBody] Meal m)
+    {
+        _repo.AddMeal(m);
+    }
```

Comment:
In the `IMealRepository` I can see that the method is using `Task`, therefore this endpoint should also be using `Task` instead of `void` and `await _repo.AddMeal(m)`

Reply:
done